packages/utils/upgrade/resources/codemods/5.0.0/utils-public-interface.code.ts

```diff
@@ -111,14 +111,18 @@ const transformImports = (root: Collection, j: JSCodeshift) => {
       source: { value: '@strapi/utils' },
     })
     .forEach((path) => {
-      path.value.specifiers?.forEach((specifier) => {
+      if (!j.ImportDeclaration.check(path.value)) {
+        return;
+      }
+
+      path.value.specifiers.forEach((specifier) => {
         if (!j.ImportSpecifier.check(specifier)) {
           return false;
         }
 
         if (removed.includes(specifier.imported.name)) {
           console.warn(
-            `Function "${specifier.imported.name}" as removed. You will have to remove it from your code.`
+            `Function "${specifier.imported.name}" was removed. You will have to remove it from your code.`
```

Comment:
I believe there is at least another instance of this in the file

Reply:
I'll check